lib/recurly/pricing/index.js

```diff
@@ -326,20 +337,19 @@ Pricing.prototype.currency = function (code, done) {
   }, this).nodeify(done);
 };
 
-Pricing.prototype.updateFactory = function (name, object) {
-  var self = this;
-
-  return function (resolve, reject) {
-    if (JSON.stringify(object) === JSON.stringify(self.items[name])) {
-      return resolve(self.items[name]);
-    }
-
-    self.items[name] = object;
-
-    debug(`set.${name}`);
-    self.emit(`set.${name}`, object);
-    resolve(object);
-  };
+/**
+ * Emits an namespaced errors
+ * @param  {Error}    error
+ * @param  {Function} reject     Promise rejection function
+ * @param  {[String]} namespace  dot-delimited error namespace
+ * @private
+ */
+Pricing.prototype.error = function (error, reject, namespace) {
+  if (namespace) {
+    namespace.split('.').reduce((memo, name) => this.emit(`${memo}.${name}`, error), 'error');
+  }
+  this.emit('error', error);
```

Comment:
This function is largely to add this event emitting without having to sprinkle the extra line all over the pricing class.